nil/services/journald_forwarder/clickhouse.go

```diff
@@ -0,0 +1,161 @@
+package journald_forwarder
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"github.com/NilFoundation/nil/nil/common/logging"
+)
+
+const (
+	DefaultDatabase = "nil"
+	DefaultTable    = "events"
+)
+
+type ClickhouseConfig struct {
+	ClickhouseAddr string
+	ListenAddr     string
+	DbUser         string
+	DbDatabase     string
+	DbPassword     string
+}
+
+type Clickhouse struct {
+	ClickhouseConfig
+	connect clickhouse.Conn
+}
+
+func (s *Clickhouse) InsertColumnsInTable(ctx context.Context,
+	database, tableName string,
+	columns, columnsType []string,
```

Comment:
Let's avoid this argument formatting, where some of it is on different lines and some of it is on the same line?

Reply:
Okay! Move ctx to newline.
But arguments with same type is looks pretty good on the same line.